src/html5sortable.ts

```diff
@@ -566,6 +568,7 @@ export default function sortable (sortableElements, options) {
 
     _data(sortableElement, 'items', options.items)
     placeholders.push(placeholder)
+    placeholderMap.set(sortableElement, placeholder)
```

Comment:
Why are you adding the Map? Is it to more easily get the placeholder? Is this necessary for this PR? Otherwise how about a different PR where you replace `placeholders` with a map? Would that work? I really like the idea of using the map, but it currently feels like it doubles a bit with `placeholders`. Please let me know if I am wrong/missing something.

Reply:
You are right, it's double work. placeholders should be replaced with a map, because it allow access to the placeholder inside of any eventhandler. I wasn't familiar enough with the placeholders so I didn't want to break anything. Definitely can be a different PR.